OpenDreamRuntime/Procs/Native/DreamProcNativeRoot.cs

```diff
@@ -2470,30 +2470,54 @@ public static DreamValue NativeProc_sorttext(NativeProc.State state)
             return new DreamValue(state.ResourceManager.SaveTextToFile(file, text) ? 1 : 0);
         }
 
+        private static int? StringToInteger(string value, int radix) {
```

Comment:
BYOND also supports floats with a radix (`text2num("F.A", 16)` is 15.625)

This method should be moved to `DreamProcNativeHelpers.cs`

Reply:
Alright. I've done more complex logic, which seems closer to BYOND implementation.